test/functional/wallet_avoidreuse.py

```diff
@@ -191,45 +199,45 @@ def test_fund_send_fund_send(self):
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 1 single, unused 10 btc output
+        # listunspent should show 1 single, unused 10 btc output.
         assert_unspent(self.nodes[1], total_count=1, total_sum=10, reused_supported=True, reused_count=0)
-        # getbalances should show no used, 10 btc trusted
+        # getbalances should show no used, 10 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 0, "trusted": 10})
 
         self.nodes[1].sendtoaddress(retaddr, 5)
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 1 single, unused 5 btc output
+        # listunspent should show 1 single, unused 5 btc output.
         assert_unspent(self.nodes[1], total_count=1, total_sum=5, reused_supported=True, reused_count=0)
-        # getbalances should show no used, 5 btc trusted
+        # getbalances should show no used, 5 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 0, "trusted": 5})
 
         self.nodes[0].sendtoaddress(fundaddr, 10)
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 2 total outputs (5, 10 btc), one unused (5), one reused (10)
+        # listunspent should show 2 total outputs (5, 10 btc), one unused (5), one reused (10).
         assert_unspent(self.nodes[1], total_count=2, total_sum=15, reused_count=1, reused_sum=10)
-        # getbalances should show 10 used, 5 btc trusted
+        # getbalances should show 10 used, 5 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 10, "trusted": 5})
 
-        # node 1 should now have a balance of 5 (no dirty) or 15 (including dirty)
+        # Node 1 should now have a balance of 5 (no dirty) or 15 (including dirty).
         assert_approx(self.nodes[1].getbalance(), 5, 0.001)
         assert_approx(self.nodes[1].getbalance(avoid_reuse=False), 15, 0.001)
 
         assert_raises_rpc_error(-6, "Insufficient funds", self.nodes[1].sendtoaddress, retaddr, 10)
 
         self.nodes[1].sendtoaddress(retaddr, 4)
 
-        # listunspent should show 2 total outputs (1, 10 btc), one unused (1), one reused (10)
+        # listunspent should show 2 total outputs (1, 10 btc), one unused (1), one reused (10).
         assert_unspent(self.nodes[1], total_count=2, total_sum=11, reused_count=1, reused_sum=10)
-        # getbalances should show 10 used, 1 btc trusted
+        # getbalances should show 10 used, 1 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 10, "trusted": 1})
 
-        # node 1 should now have about 1 btc left (no dirty) and 11 (including dirty)
+        # Node 1 should now have about 1 btc left (no dirty) and 11 (including dirty).
         assert_approx(self.nodes[1].getbalance(), 1, 0.001)
         assert_approx(self.nodes[1].getbalance(avoid_reuse=False), 11, 0.001)
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
Why is this changed. Most other tests use single quotes. In python both serve the same function and I don't see the need to change this.

Reply:
Was being consistent with the rest of this test which is mostly in double quotes and with https://www.python.org/dev/peps/pep-0008/#documentation-strings